### PR TITLE
run_udf(_externally): don't mention raster-cube in the schemas

### PR DESCRIPTION
**Pull request by m-mohr**
See issue #285 for context.

This was discussed today with some of you already and this is the resulting PR.

### CODE REVIEW

File: proposals/run_udf_externally.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
-    "description": "Runs a compatible UDF container that is either externally hosted by a service provider or running on a local machine of the user. The UDF container must follow the [openEO UDF specification](https://openeo.org/documentation/1.0/udfs.html).\n\nThe referenced UDF service can be executed in several processes such as ``aggregate_spatial()``, ``apply()``, ``apply_dimension()`` and ``reduce_dimension()``. In this case, an array is passed instead of a raster data cube. The user must ensure that the data is properly passed as an array so that the UDF can make sense of it.",
+    "description": "Runs a compatible UDF container that is either externally hosted by a service provider or running on a local machine of the user. The UDF container must follow the [openEO UDF specification](https://openeo.org/documentation/1.0/udfs.html).\n\nThe referenced UDF service can be executed in several processes such as ``aggregate_spatial()``, ``apply()``, ``apply_dimension()`` and ``reduce_dimension()``. In this case, an array is passed instead of a raster data cube. The user must ensure that the data is passed over in a way that the UDF code can make sense of it.",
```
Comment (soxofaan) [inline proposals/run_udf_externally.json:4]: > ... the data is passed over in a way ...

is "pass over" proper English?

I think just "pass" works fine

File: run_udf.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+    "description": "Runs a UDF in one of the supported runtime environments.\n\nThe process can either:\n\n1. load and run a UDF stored in a file on the server-side workspace of the authenticated user. The path to the UDF file must be relative to the root directory of the user's workspace.\n2. fetch and run a remotely stored and published UDF by absolute URI.\n3. run the source code specified inline as string.\n\nThe loaded UDF can be executed in several processes such as ``aggregate_spatial()``, ``apply()``, ``apply_dimension()`` and ``reduce_dimension()``. The user must ensure that the data is passed over in a way that the UDF code can make sense of it.",
```
Comment (soxofaan) [inline run_udf.json:4]: (see higher about "pass over")

**Review by soxofaan — APPROVED** [review]
looks fine to me, 
just a minor note

**Review by przell — APPROVED** [review]
I've commented directly. Generally also like this approach!

File: run_udf.json (hunk 2 above)
Comment (m-mohr) [reply run_udf.json:4]:
```suggestion
    "description": "Runs a UDF in one of the supported runtime environments.\n\nThe process can either:\n\n1. load and run a UDF stored in a file on the server-side workspace of the authenticated user. The path to the UDF file must be relative to the root directory of the user's workspace.\n2. fetch and run a remotely stored and published UDF by absolute URI.\n3. run the source code specified inline as string.\n\nThe loaded UDF can be executed in several processes such as ``aggregate_spatial()``, ``apply()``, ``apply_dimension()`` and ``reduce_dimension()``. The user must ensure that the data is given in a way that the UDF code can make sense of it.",
```

File: proposals/run_udf_externally.json [hunk 3]
```diff
@@ -53,18 +48,11 @@
...
-                "title": "Raster data cube",
-                "type": "object",
-                "subtype": "raster-cube"
-            },
-            {
-                "title": "Any",
-                "description": "Any data type."
-            }
-        ]
+        "description": "The data processed by the UDF. The returned value can be of any data type and is exactly what the UDF code returns.",
```
[the remark this answers is not in the document]
Comment (m-mohr) [reply proposals/run_udf_externally.json:51]:
```suggestion
        "description": "The data processed by the UDF. The returned value can in principle be of any data type, but it depends on what is returned by the UDF code. Please see the implemented UDF interface for details.",
```

[the remark this answers is not in the document]
Comment (m-mohr) [reply proposals/run_udf_externally.json:51]: UDF interface here refers to the "UDF signature" implemented as there are a couple of helpers that make analysis easier for the user, but influence what is returned.

**Pull request merged**